packages/aws-cdk-lib/aws-ec2/README.md

```diff
@@ -1726,6 +1726,39 @@ in the `cdk.json` file.
 }
 ```
 
+### Transit Gateway
+
+You also can create a flow log for a transit gateway.
+
+```ts
+const transitGateway = new ec2.CfnTransitGateway(this, 'TransitGateway', {});
+new ec2.FlowLog(this, 'FlowLogsCW', {
+  resourceType: ec2.FlowLogResourceType.fromTransitGatewayId(transitGateway.ref),
+  flowLogName: 'TransitGatewayFlowLogName',
+});
+```
+
+You can create from a transit gateway attachment.
+
+```ts
+const transitGateway = new ec2.CfnTransitGateway(this, 'TransitGateway', {});
+const transitGatewayAttachment = new ec2.CfnTransitGatewayVpcAttachment(
+  this,
+  'TransitGatewayAttachment',
+  {
+    subnetIds: vpc.selectSubnets({
```

Comment:
This is causing the build to fail since you have not created the vpc. Add the vpc declaration to the example:
`declare const vpc: ec2.Vpc;`

Reply:
Thank you. I worked on it.